spotify_tensorflow/luigi/tfx_task.py

```diff
@@ -52,6 +55,10 @@ def tfx_args(self):
             "--schema_file=%s" % self.get_schema_file()
         ]
 
+    def dataflow_executable(self):
+        """ Must be overwritten from the BeamDataflowTaski """
```

Comment:
typo?

Reply:
Yup, good catch

PR is still WIP cause the new beam task is refactored a bunch, trying to fix tests